main.go

```diff
@@ -94,6 +94,22 @@ func main() {
 		setupLog.Error(err, "unable to create controller", "controller", "ClusterSecret")
 		os.Exit(1)
 	}
+
+	if err = (&controllers.ValueFromSecretReconciler{
+		Client: mgr.GetClient(),
+		Scheme: mgr.GetScheme(),
+	}).SetupWithManager(mgr); err != nil {
+		setupLog.Error(err, "unable to create controller", "controller", "ValueFromSecret")
+		os.Exit(1)
+	}
+
```

Comment:
nit: You can also construct log into the manager :) 
Ex: https://github.com/druid-io/druid-operator/blob/master/controllers/druid/druid_controller.go#L28

Reply:
Again, thanks for the suggestion, let's create an issue to track this in the next version ;)